spec/01-lexical-syntax.md

```diff
@@ -47,7 +47,7 @@ hexDigit      ::= ‘0’ | … | ‘9’ | ‘A’ | … | ‘F’ | ‘a’ |
 ```
 
 There are three ways to form an identifier. First, an identifier can
-start with a letter which can be followed by an arbitrary sequence of
+start with a letter that can be followed by an arbitrary sequence of
```

Comment:
perhaps better: "start with a letter, followed by"

Reply:
Agreed. Changed it.